game.cpp

```diff
@@ -1134,6 +1134,25 @@ void test_game() //!OCLINT tests may be many
   }
 #endif
 
+//#define FIX_ISSUE_463
+#ifdef FIX_ISSUE_463
```

Comment:
After the tests are done, remove the preprocessor statements (i.e. `#define`, `#ifdef`, `#endif`). They are there to put tests in the code that fail (so our Issue can point to them). When the tests passes, these can be removed.

I do usually make it a newbie Issue to remove these, so maybe that is your strategy as well :-)